packages/11ty/_layouts/entry.liquid

```diff
@@ -46,18 +46,28 @@ Entry content, including entry image and tombstone data
         {% elsif pageObjects %}
           {% for object in pageObjects %}
             <div class="quire-entry__image__group-container">
-              {% assign figureLength = object.figure | size %}
-              {% if figureLength > 1 and pdf != true -%}
-                <button class="remove-from-epub quire-image-control quire-image-control--prev-image" id="prev-image" title="Previous image"></button>
-                <button class="remove-from-epub quire-image-control quire-image-control--next-image" id="next-image" title="Next image"></button>
-              {% endif -%}
               {% for objectFigure in object.figures %}
                 {% render "entry-figure", epub: epub, pdf: pdf, zoom: zoom, imageDir: imageDir, figure: objectFigure %}
               {% endfor %}
             </div>
           {% endfor %}
         {% endif %}
       </div>
+      {% comment %}
+        TODO refactor `entry-figure` to only display on PDF/epub output so `lightbox` can handle display of all entry figures in the browser.
+        Currently this is (hackily) displaying a `lightbox` on top of rendered `entry-figure`s
+      {% endcomment %}
+      <div class="quire-entry__lightbox remove-from-epub">
+        {% if image %}
+          {%- assign imgPath = imageDir %}
+          {%- assign imgPath = imgPath | concat: image | join: "/" %}
+          {% comment %} TODO add `lightbox` when a single `image` is included in front matter {% endcomment %}
+        {% elsif pageObjects %}
+          {% for object in pageObjects %}
+            {% lightbox object.figures %}
+          {% endfor %}
+        {% endif %}
+      </div>
```

Comment:
The `lightbox` is being added IN ADDITION TO rendering all `entry-figure`s  above. `entry-figure` needs to likely be refactored to only display in PDF/epub output so`lightbox` can handle universal display of all figure types.